share/spice/symbolab/subtitle_content_nolatex.handlebars

```diff
@@ -0,0 +1 @@
+Solution for: <span>{{latex}}</span>
```

Comment:
I think it would be better to use an `{{if}}` block to add the Mathquill class when needed.

It could look like this:

```js
normalize: function(data){
  return {
    hasLatex: containsLatex(data.solutionLatex),
    solution: data.solution,
    ...
  }
}
```
```handlebars
Solution for: <span {{if hasLatex}}clas="mathquill-embeded-latex"{{/if}}>{{solution}}</span>
```

This should simplify the JS and also reduce the number of templates we're using.